samza-rest/src/main/java/org/apache/samza/monitor/LocalStoreMonitorConfig.java

```diff
@@ -53,6 +53,12 @@
    */
   private static final long DEFAULT_OFFSET_FILE_TTL_MS = 1000 * 60 * 60 * 24 * 7;
 
+  /**
+   * Boolean variable if set to true will make LocalStoreMonitor to ignore failures
```

Comment:
Don't need 'Boolean variable'. Esp. since the actual type is string.

'will make LocalStoreMonitor ignore'